config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,38 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.2 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
```

Comment:
have you made the flips yet? or do we have to do that after?

Reply:
@akegan I did them one by one (ish) locally so they're all already flipped except for `Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = true` which should only be flipped once you're stable on 5.2, so I'm not in any rush to turn that on in a hotfix.

I did update the comment to remove the "Once upgraded flip defaults one by one" line since it's confusing to leave in after we're already upgraded and the flags are flipped

Reply:
ok sounds reasonable